google/api_core/grpc_helpers.py

```diff
@@ -297,6 +301,9 @@ def create_channel(
         default_scopes (Sequence[str]): Default scopes passed by a Google client
             library. Use 'scopes' for user-defined scopes.
         default_host (str): The default endpoint. e.g., "pubsub.googleapis.com".
+        compression (grpc.Compression): An optional value indicating the
+            compression method to be used over the lifetime of the channel.
+            This is an EXPERIMENTAL option.
```

Comment:
Can we remove the experimental disclaimer? If not, can you clarify the reason that it is experimental in the comment with a note about when we can remove the experimental disclaimer?